.github/workflows/cd_wiki.yaml

```diff
@@ -0,0 +1,46 @@
+name: Wiki CD
+
+on:
+  workflow_dispatch:
```

Comment:
Is this only going to run manually? Since the code below works if there are no changes at all, we could run this after every commit, or even on schedule.

Reply:
How often do you think it should run?
In either case, I think we would still want the option to run it manually, and I think it would be simpler if there's only one way that it is run.

Reply:
I'd say running it daily should be more than enough. It doesn't take too long.